oscarapi/utils.py

```diff
@@ -17,11 +17,13 @@ def overridable(name, default):
     return getattr(settings, name, default)
 
 
+field_mapping = serializers.ModelSerializer.serializer_field_mapping
+field_mapping.update({
+    oscar.models.fields.NullCharField: serializers.CharField
+})
```

Comment:
If you are bothered by how this looks in class scope, define a function that does it for you:

```
def expand_field_mapping(**extra_fields):
    field_mapping = serializers.ModelSerializer.serializer_field_mapping  # btw this doesn't make a copy, so you are modifying a class variable in serializers.ModelSerializer here
    field_mapping.update(extra_fields)
    return field_mapping